docs/guides/qiskit-addons-cutting-wires.ipynb

```diff
@@ -0,0 +1,552 @@
+{
```

Comment:
<blockquote>Cut wires automatically</blockquote><p>I suggest: Cut wires using high-level <code>CutWire</code> instructions</p><p> </p><p>and below: Cut wires using low-level <code>Move</code> instructions </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>